crates/ethereum/payload/src/lib.rs

```diff
@@ -79,7 +86,11 @@ where
         &self,
         args: BuildArguments<Pool, Client, EthPayloadBuilderAttributes, EthBuiltPayload>,
     ) -> Result<BuildOutcome<EthBuiltPayload>, PayloadBuilderError> {
-        default_ethereum_payload_builder(self.evm_config.clone(), args)
+        default_ethereum_payload_builder(
+            self.evm_config.clone(),
+            args,
+            NoOpTransactionFilter::new(),
+        )
```

Comment:
I'd like to tackle this feature differently, by giving the

https://github.com/paradigmxyz/reth/blob/5fc77551dde64769f911fbc6c9d9008a380e4ecd/crates/ethereum/payload/src/lib.rs#L52-L52

an instance of a filter